pkg/reconciler/ingress/ingress.go

```diff
@@ -249,7 +249,12 @@ func (r *BaseIngressReconciler) reconcileIngress(ctx context.Context, ra Reconci
 	logger.Infof("Reconciling ingress: %#v", ia)
 
 	gatewayNames := gatewayNamesFromContext(ctx, ia)
-	vses := resources.MakeVirtualServices(ia, gatewayNames)
+	istioEnabled, err := isIstioInjectionEnabledForNs(r, ra, ia)
+	if err != nil {
+		logger.Errorw("Failed to retrieve namespace label %v", err)
```

Comment:
`Errorw` requires `zap.Error`.
Also which label? Perhaps we can log in the method itself.